Packages/PrimitivesComponents/Sources/Components/InputValidationField.swift

```diff
@@ -3,13 +3,15 @@
 import SwiftUI
 import Components
 import Style
+import Primitives
 
 public struct InputValidationField<TrailingView: View>: View {
     @Binding private var model: InputValidationViewModel
 
     let style: FloatFieldStyle
     let placeholder: String
     let allowClean: Bool
+    let onWillClean: VoidAction
```

Comment:
```suggestion
    let onCleanTextfield: VoidAction
```

or something more intuitive